lib/common/Makefile.am

```diff
@@ -28,7 +28,7 @@ SUBDIRS = tests
 
 noinst_HEADERS		= crmcommon_private.h
 
-libcrmcommon_la_LDFLAGS	= -version-info 37:0:3
+libcrmcommon_la_LDFLAGS	= -version-info 38:0:0
```

Comment:
What are you considering a backward-compatible change here?

Reply:
pcmk_copy_time added to iso8601.h
new conversion functions added to results.h (+ code relying on getting the new return codes?)

Reply:
Additions are backward-compatible, so 38:0:4 is fine. (38 == this is a new set of interfaces, 4 == but it's compatible 4 revisions back).

The new return code is perhaps borderline. New code relying on the new return codes is fine, that's backward compatible -- the only question would be old code relying on old return codes where the new ones are now returned. But as a practical matter we don't have to worry about that. Any external code using the new interfaces is handled by forward-compatibility of older libraries, i.e. external code linked against the new library version will fail to load with older versions. But external code using the old interfaces can upgrade to the new library with no problem.

If old code can use the new library -> C+1:0:A+1; if old code would break -> C+1:0:0.

Reply:
Fully agreed. Don't remember what made me go for incompatible first.
Actually I hadn't even realized that I had done so. Your question was what made me consider backward-compatible ;-)